rts/lra/lra-coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java

```diff
@@ -381,44 +381,28 @@ public Response joinLRAViaBody(
                     + " complete, compensate and status.",
                     required = false )
             @HeaderParam("Link") String compensatorLink,
-            @ApiParam( value = "The resource path that the LRA coordinator will use to drive the participant.\n"
-                    + "Performing a GET on the participant URL will return the current status of the participant,\n"
-                    + "or 404 if the participant is no longer present.\n"
-                    + "\n"
-                    + "The following types must be returned by Compensators to indicate their current status:\n"
-                    + "-  Compensating: the Compensator is currently compensating for the jfdi.\n"
-                    + "-  Compensated: the Compensator has successfully compensated for the jfdi.\n"
-                    + "-  FailedToCompensate: the Compensator was not able to compensate for the jfdi.\n"
-                    + "   It must maintain information about the work it was to compensate until the\n"
-                    + "   coordinator sends it a forget message.\n"
-                    + "-  Completing: the Compensator is tidying up after being told to complete.\n"
-                    + "-  Completed: the coordinator/participant has confirmed.\n"
-                    + "-  FailedToComplete: the Compensator was unable to tidy-up.\n"
-                    + "\n"
-                    + "Performing a PUT on <URL>/compensate will cause the participant to compensate\n"
-                    + "  the work that was done within the scope of the LRA.\n"
-                    + "Performing a PUT on <URL>/complete will cause the participant to tidy up and\n"
-                    + "   it can forget this LRA.\n")
-                    String compensatorUrl) throws NotFoundException {
+            @ApiParam( value = "data that will be stored with the coordinator and passed back to\n"
```

Comment:
I think it is worth pointing out this might be a URL though right?

Reply:
I added the word opaque as in "opaque data that will be stored ...", ie it can be anything the participant deems fit for its own purposes.

Reply:
sure - what I figured is that our implementation has special handling of this otherwise opaque data that might be useful for a user to know, e.g. on line 399 if the data happens to be a URL we will assume it is the participant URL

Reply:
Ah I see what you are saying. The original name of "compensatorUrl" and the swagger description was incorrect so I changed it - ie it wasn't a URL it was opaque data meaningful only to the participant.